src/v2/view-builder/views/device/DeviceCodeActivateView.js

```diff
@@ -1,6 +1,39 @@
-import { loc } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+import { loc, View } from 'okta';
 import { BaseForm, BaseView } from '../../internals';
 
+const InvalidUserCodeErrorView = View.extend({
```

Comment:
you don't need to create a new view for this to show errors. It's handled by widget's `showMessage` method if you override it.

take a look at `EnrollPollOktaVerifyView` on how it does override `showMessages`

Reply:
good tip, thanks. I changed it.